### PR TITLE
Add Support for optional additional property

### PR DESCRIPTION
**Pull request by MarkDordoy**
Adding `max_size_limit` as an optional property. When this is used, it allows the groups added to the token or SAML response to go from 150 to a max of 1000.

### CODE REVIEW

File: internal/services/applications/migrations/application_resource.go [hunk 1]
```diff
@@ -328,6 +328,7 @@ func ResourceApplicationInstanceResourceV0() *schema.Resource {
 													"emit_as_roles",
 													"include_externally_authenticated_upn",
 													"include_externally_authenticated_upn_without_hash",
+												        "max_size_limit",
 													"netbios_domain_and_sam_account_name",
```
Comment (manicminer) [inline internal/services/applications/migrations/application_resource.go:332]: tabs vs spaces :)

```suggestion
													"include_externally_authenticated_upn_without_hash",
													"max_size_limit",
													"netbios_domain_and_sam_account_name",
```

**Review by katbyte — APPROVED** [review]
LGTM 🌳

File: internal/services/applications/migrations/application_resource.go [hunk 2]
```diff
@@ -374,6 +375,7 @@ func ResourceApplicationInstanceResourceV0() *schema.Resource {
 													"emit_as_roles",
 													"include_externally_authenticated_upn",
 													"include_externally_authenticated_upn_without_hash",
+													"max_size_limit",
```
Comment (manicminer) [inline internal/services/applications/migrations/application_resource.go:378]: On a second pass, we don't need to add this here as this should be a representative snapshot of the schema available at that time.
```suggestion
```

File: internal/services/applications/migrations/application_resource.go [hunk 3]
```diff
@@ -328,6 +328,7 @@ func ResourceApplicationInstanceResourceV0() *schema.Resource {
...
```
Comment (manicminer) [inline internal/services/applications/migrations/application_resource.go:331]: Same here
```suggestion
```

**Pull request merged**